@shared/helpers/stellar.ts

```diff
@@ -42,93 +46,99 @@ export function getBalanceIdentifier(
   }
 }
 
+export const defaultBlockaidScanAssetResult: BlockAidScanAssetResult = {
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  result_type: "Benign",
+  features: [{ description: "" }],
+};
+
 export function makeDisplayableBalances(
   accountDetails: StellarSdk.Horizon.ServerApi.AccountRecord,
-): BalanceMap {
+) {
   const { balances, subentry_count, num_sponsored, num_sponsoring } =
     accountDetails;
 
-  const displayableBalances = Object.values(balances).reduce(
```

Comment:
using `reduce` made it hard for TS to typecheck the different balance types. Switching to a for loop for more clarity.

Also, at some point `balances` must have changed from an object to an array. No need to do this `Object.values` conversion